### ISSUE TITLE
Does it support wfs and wms?

### ISSUE THREAD

**Issue by giszh86** [Terradue/DotNetOgcModel#2]
Does it support wfs and wms?

**Comment by emmanuelmathot** [comment]
This library does not implement the Web Services. It contains all the data model ready for (de)serialization.

**Comment by giszh86** [comment]
Yes,I know,but wfs and wms also have data model.

**Comment by emmanuelmathot** [comment]
Yes indeed. The enclosing type for WMS and WFS are not implemented. Have a look at WCS or WPS (https://github.com/Terradue/DotNetOgcModel/blob/develop/Terradue.ServiceModel.Ogc/Terradue/ServiceModel/Ogc/Wps10) implementation for a design pattern. All the data types necessary in WMS and WFS and included in this library. For instance the WMS GetCapabilities is similar to the WPS one ((https://github.com/Terradue/DotNetOgcModel/blob/develop/Terradue.ServiceModel.Ogc/Terradue/ServiceModel/Ogc/Wps10/Capabilities.cs). Only the underlying handlers would change.
